ui/jobs/jobs_config.ts

```diff
@@ -219,5 +219,26 @@ UserConfig.registerOptions('jobs', {
       type: 'integer',
       default: -1,
     },
+    {
+      id: 'NotifyExpiredProcsInCombatSound',
+      name: {
+        en: 'Play a sound notification if a proc box for dots/etc. expires while in combat.',
+        de: 'Spiele einen Alarm Sound wenn eine dot/etc. box im Kampf inaktiv wird.',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'Disabled': 'disabled',
+          'When counter reaches 0': 'expired',
+          'When counter reaches threshold': 'threshold',
```

Comment:
I don't like this description as I'm unsure how many users actually know what is meant by threshold. Would appreciate any suggestions.

Reply:
"When counter gets close to zero" or "When the proc box gets big when nearly expired" or something along those lines?